content/department-processes/mock-interviews/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+ready: true
+title: Mock Interviews
+---
+
+Mock Interviews are done so to give recruits a glimpse of what they can expect from a real
+life interview. This is a great way to prepare them of what is to come.
+
+We have compiled a couple of questions that we know are likely asked in a real life interview.
+
+This document has those questions and are to be used when conducting a Mock Interview.
+https://docs.google.com/document/d/1O6IGatGF-rUZsMO3ORwVdIigLF4Cgx7-MKsmMQXd4y8/edit
+
+These interviews not only involve the questions but the following too;
+### Mock Interview Process
+
+Mock Interviews are done so to give you a glimpse of what you can expect from a real
+life interview. This is a great way to prepare you of what is to come.
+
+The Mock Interviews also invlove the following as we try to make it as real life as possible.
+
+ - Whitebording
```

Comment:
Ok. no. You just cleared my comments, you did not address them.

I asked for more detail not less. I dont know what you are trying to do here.

What is whitebaording? 
How does one prepare for whiteboarding.

The last version of this file was better than this version. Please add the missing detail.

Aim to make it more useful. Dont aim to make it less useful